src/css/style.css

```diff
@@ -1,3 +1,4 @@
 @import "reset.css";
 @import "colors.css";
-@import "base.css";
\ No newline at end of file
+@import "base.css";
+@import "brand.css";
```

Comment:
The name of this file should be improved, it handles the header stuff so it should be called header. Doing so will make it clearer what his responsibility is and all the style that needs to be defined in the header will be in this file.